src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,364 @@
+  }
+}
+
+.pf-c-sequence__control-left {
+  left: var(--pf-c-sequence__control-left--PositionLeft);
+  background-color: var(--pf-c-sequence__control-left--BackgroundColor);
+  border-right: var(--pf-c-sequence__control-left--BorderRightWidth) solid var(--pf-c-sequence__control-left--BorderRightColor);
+}
+
+.pf-c-sequence__control-right {
+  right: var(--pf-c-sequence__control-right--PositionRight);
+  background-color: var(--pf-c-sequence__control-right--BackgroundColor);
+  border-left: var(--pf-c-sequence__control-right--BorderLeftWidth) solid var(--pf-c-sequence__control-right--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  overflow: scroll;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+
+  // added so that the list will be read by screen reader
+  &::before {
+    width: 1px;
+    content: "";
+  }
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+  margin-right: var(--pf-c-sequence__item--MarginRight);
+  margin-left: var(--pf-c-sequence__item--MarginLeft);
+}
+
+// Vertically centers number and border
+.pf-c-sequence__number-container {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+// Sequence Link Position Number
+.pf-c-sequence__number {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-width: var(--pf-c-sequence__number--MinWidth);
+  max-height: var(--pf-c-sequence__number--MaxHeight);
+  padding: var(--pf-c-sequence__number--Padding);
+  margin-right: var(--pf-c-sequence__number--MarginRight);
+  font-size: var(--pf-c-sequence__number--FontSize);
+  color: var(--pf-c-sequence__number--Color);
+  text-align: center;
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-sequence__number--BorderWeight) solid var(--pf-c-sequence__number--BorderColor);
+    border-radius: var(--pf-c-sequence__number--BorderRadius);
+  }
+}
+
+// Content
+.pf-c-sequence__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: var(--pf-c-sequence__content--MinHeight);
+}
+
+// Content Title
+.pf-c-sequence__content-title {
+  display: block;
+  max-width: var(--pf-c-sequence__content--title--MaxWidth);
+  overflow: hidden;
+  font-weight: var(--pf-c-sequence__content-title--FontWeight);
+  color: var(--pf-c-sequence__content-title--Color);
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+// Content Status
+.pf-c-sequence__content-status {
+  font-size: var(--pf-c-sequence__content-status--FontSize);
+  color: var(--pf-c-sequence__content-status--Color);
```

Comment:
If you put a lot of text in this element, it pushes the parent link's border down.

Reply:
Do you think a good option would be to truncate it? Since its a status, im assuming that means it shouldnt really be more than a few words

Reply:
@michael-coker

Reply:
That seems fine to me, since you're truncating the title text, too. You can always ask one of the designers as well how they expect the element to behave with more/less text.

Reply:
truncated for now, will double check with Kyle when he reviews